helmfile/values/userCRDs/common.yaml.gotmpl

```diff
@@ -0,0 +1 @@
+namespaces: {{ .Values.user.namespaces }}
```

Comment:
```suggestion
namespaces: {{ toYaml .Values.user.namespaces | nindent 2 }}
```

Reply:
Right it is a list. I have just worked with one namespace :upside_down_face: